### PR TITLE
Issue#150

### PR DESCRIPTION
**Pull request by ihordubas1**
#150

### CODE REVIEW

**Comment by ihordubas1** [comment]
Hello @kfarr 
should be fine now

**Comment by kfarr** [comment]
thanks @ihordubas1 I checked it out and it looks good so far. Next I'll review the code but it will have to be tomorrow, getting late

File: src/components/components/Component.js [hunk 1]
```diff
@@ -117,16 +117,18 @@ export default class Component extends React.Component {
...
-          key={propertyName}
-          name={propertyName}
-          schema={componentData.schema[propertyName]}
-          data={componentData.data[propertyName]}
-          componentname={this.props.name}
-          isSingle={false}
-          entity={this.props.entity}
-        />
+      .map((propertyName, idx) => (
+        <div className="detailed" key={idx}>
```
Comment (kfarr) [inline src/components/components/Component.js:121]: * It doesn't appear classname `detailed` is used?
* Curious why `idx` is needed here?
* If classname is not used and idx is not used, then maybe this div not needed at all?

Comment (ihordubas1) [reply src/components/components/Component.js:121]: No, this div is needed to highlight the bottom part of the settings block. This classname is just used elsewhere to make things work properly, so there’s no point in deleting anything here.
As for idx, react will resent if you don’t pass the keys in the map method.

Comment (kfarr) [reply src/components/components/Component.js:121]: thanks @ihordubas1 !

**Pull request merged**